modules/payment/src/PluginForm/OffsitePaymentForm.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Drupal\commerce_payment\PluginForm;
+
+use Drupal\commerce_payment\Plugin\Commerce\PaymentGateway\OffsiteRedirectPaymentGatewayInterface;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Render\Element;
+
+abstract class OffsitePaymentForm extends PaymentGatewayFormBase {
```

Comment:
This form is usable out of the box, no? Cause it just sends you to the redirect url?
We still aren't covering all three use cases (direct redirect, form redirect, iframe), for which we probably want 3 different classes.